pkg/extender/extender.go

```diff
@@ -366,6 +368,18 @@ func (e *Extender) volumePrefersRemoteOnly(volumeInfo *volume.Info) bool {
 	return false
 }
 
+// volumePrefersRemote checks if preferRemoteNode label is applied to the volume, else returns default value True.
+func (e *Extender) volumePrefersRemote(volumeInfo *volume.Info) bool {
```

Comment:
nit.. Let's change the name to volumePrefersRemoteNode.
I know we have another function with name volumePrefersRemoteOnly and that's why you chose this name. 
so let's change volumePrefersRemoteOnly too to use volumePrefersRemoteNodeOnly.
Current function name is not very clear in regards to Remote What?

Reply:
Makes sense. Will address in the revision